lib/honeybadger.ex

```diff
@@ -299,9 +300,17 @@ defmodule Honeybadger do
       |> List.to_string()
     end
 
+    git_revision = fn ->
```

Comment:
If `git` isn't available on the system this will raise `** (ErlangError) Erlang error: :enoent`. We should use [find_executable/1](https://hexdocs.pm/elixir/System.html#find_executable/1) to check for the executable before making the call.

Reply:
@sorentwo Oh I see yes, I just noticed that it would throw an actual error then yes